tests/core/sdirk_stage.cc

```diff
@@ -0,0 +1,67 @@
+// SPDX-FileCopyrightText: Copyright (c) 2019-2020, 2024 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#include <core/sdirk_stage_data.h>
+#include <core/sdirk_table.h>
+
+// Tests (with common definitions)
+#include <../tests/tests.h>
+
+void
+test()
+{
+  deallog << "Testing SDIRK2 coefficients" << std::endl;
```

Comment:
I would also add a test for SDIRK33 coefficients. Maybe also print with more significant digits than here? It might be worth it to print with like 12 digits of accuracy to make sure that you really are calculaating what you think :)